packages/lib/modules/pool/PoolDetail/PoolStats/PoolSnapshot/PoolSnapshot.tsx

```diff
@@ -40,20 +39,16 @@ const TABS = [
 ] as const
 
 export function PoolSnapshot({ ...props }: CardProps) {
-  const [activeTab, setActiveTab] = useState<ButtonGroupOption>(TABS[0])
   const { pool } = usePool()
   const { colorMode } = useColorMode()
+  const defaultTab = hasTotalBalance(pool) ? TABS[1] : TABS[0]
```

Comment:
Don't really understand what we are trying to do here. Do you know why the changing of the default balance here depending on if we have some balance or not?